components/office-card/office-card.less

```diff
@@ -0,0 +1,143 @@
+@import (reference) '../../styles/variables';
+@import (reference) '../../styles/shadows';
+
+.office-card {
+	margin: 0 0 @spacing-large 0;
+
+	a {
+		color: @darker-gray;
+		text-decoration: none;
+	}
+
+	a:hover {
+		color: @red;
+
+		.office-card-image .image {
+			transform: scale(1.05);
+		}
+	}
+
+	@media @screen-small {
+		float: left;
+		margin-bottom: @spacing-extra-large;
+		width: calc(~'50% - ' @spacing-small);
```

Comment:
I would put all the positioning of the cards in the parent component `office-overview`. Just like the `update-overview` component